manifest_template.yml

```diff
@@ -7,25 +7,18 @@ APIGEE_ENVIRONMENTS:
     has_mock_auth: true
   - name: internal-qa
     display_name: Internal QA
-    has_mock_auth: true 
+    has_mock_auth: true
   - name: ref
     display_name: Reference
     has_mock_auth: true
 # Enable environments when ready by uncommenting:
-  # - name: internal-dev-sandbox
-  #   display_name: Internal Development Sandbox
-  #   portal_visibility: false
-  # - name: internal-qa
-  #   display_name: Internal QA
-  #   has_mock_auth: true
-  #   portal_visibility: false
-  # - name: internal-qa-sandbox
-  #   display_name: Internal QA Sandbox
-  #   portal_visibility: false
-  # - name: sandbox
-  #   display_name: Sandbox
-  #   portal_visibility: false
-  # - name: int
+  - name: internal-dev-sandbox
```

Comment:
@ChewieCB to confirm, this manifest controls the environments deployed?

Reply:
yes, it determines which apigee proxies are deployed during the PR pipeline